recipes/mapd-core-cpu/meta.yaml

```diff
@@ -0,0 +1,92 @@
+{% set name = "mapd-core-cpu" %}
+{% set version = "4.5.0" %}
+{% set sha256 = "d2a0f7ffd4366a38aed2a5a81e69afde5293aa03fdcbf5142ff2caed6cccce5b" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mapd/mapd-core/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - arrow-cpp >=0.12
+    - bisonpp
+    - blosc
+    - boost-cpp 1.68
+    - boost 1.68
+    - clangdev 6
+    - cmake
+    - cython
+    - double-conversion
+    - flex
+    - gdal
+    - gflags
+    - git
+    - glog
+    - go
+    - libarchive
+    - libkml
+    - libpng
+    - llvmdev 6
+    - make
+    - maven
+    - openjdk 8
+    - openssl
+    - pytest
+    - python >=3.6
+    - setuptools
+    - numba >=0.40
+    - numpy
+    - parquet-cpp
+    - pyarrow >=0.12
+    - thrift-cpp >=0.11.0
+    - zlib
+    - wget
```

Comment:
It seems like most of these are host, not build requirements

Reply:
you're right. not sure why .. but in the past we tried to move to host .. but it raised some weird errors ...

Reply:
Sure, but this is almost certainly the wrong thing.  Only the compilers, make, etc should be on build.  All of the libraries you link against need to be in host.

Reply:
ok I will try that, thanks!

Reply:
Also, do you really link against wget?

Reply:
no, wget is not linked against.
The minimal set of dependencies are here:
https://github.com/Quansight/pearu-sandbox/blob/master/conda-envs/omnisci-dev-ubuntu18.yaml

Reply:
OK I will remove it .. I just add it because it was installed inside mapd-core build scripts. thanks @pearu @scopatz